mne/minimum_norm/time_frequency.py

```diff
@@ -555,12 +591,13 @@ def _compute_source_psd_epochs(epochs, inverse_operator, lambda2=1. / 9.,
     except RuntimeError:
         n_epochs = len(epochs.events)
         extra = 'on at most %d epochs' % (n_epochs,)
-        pb = False
     else:
         extra = 'on %d epochs' % (n_epochs,)
-        pb = True
-    logger.info('Using %d tapers with bandwidth %0.1fHz %s'
-                % (n_tapers, bandwidth, extra))
+    if isinstance(bandwidth, string_types):
+        bandwidth = '%s windowing' % (bandwidth,)
```

Comment:
Enhancement: allow MTM estimation to take a string, meaning "use standard estimation". The advantage of this is that in the long run it will greatly simplify our code paths, especially if we go to implement CUDA-based multitaper estimation.